TeamHangman/TeamHangman/Game.cpp

```diff
@@ -151,6 +198,14 @@ void Game::print(std::string string)
 
 void Game::OnLetterPressed(std::string letter)
 {
+	int correctLetters = 0;
 	std::cout << letter << std::endl;
-	std::cout << GetRandomLine() << std::endl;
+	for (int i = 0; i < gameLetters.size(); i++)
+	{
+		if (gameLetters[i]->CompareToMyLetter(letter) == true)
+		{
+			correctLetters++;
+		}
+	}
+	// if (correctLetters == 0) {failedGuess();}
```

Comment:
gonna use this comment later?

Reply:
do not have the hangman yet to make anything happen